### PR TITLE
Progress5 updates

### PR DESCRIPTION
**Pull request by blazyy**

### CODE REVIEW

File: py-data.csv [hunk 1]
```diff
@@ -136,9 +136,9 @@ https://github.com/chaosmail/python-fs,2567922ced9387e327e65f3244caff3b7af35684,
 https://github.com/chaosmail/python-fs,2567922ced9387e327e65f3244caff3b7af35684,fs/tests/test_touch.py::test_touch_on_new_file,NIO,Accepted,https://github.com/chaosmail/python-fs/pull/9,
 https://github.com/chie8842/expstock,28b7ece83fa601efd2b6c4b8b7bdd1a3b9ebc1df,expstock/test_expstock.py::test_append_param,NIO,Opened,https://github.com/chie8842/expstock/pull/18,
 https://github.com/chinapandaman/PyPDFForm,fcc1b297005a39900c1f928f602cbc5f38a1d60e,tests/unit/test_font.py::test_register_font_and_is_registered,NIO,Accepted,https://github.com/chinapandaman/PyPDFForm/pull/254,
-https://github.com/chinapnr/fishbase,1615ccff9106bf42fc6ae862cd13353c12c1b76d,test/test_logger.py::TestFishLogging::test_format3,OD-Brit,,,
+https://github.com/chinapnr/fishbase,1615ccff9106bf42fc6ae862cd13353c12c1b76d,test/test_logger.py::TestFishLogging::test_format3,OD-Brit,MovedOrRenamed,,Test renamed to test_format_03
```
Comment (xieyt2000) [inline py-data.csv:139]: Please add new rows for the renamed tests

File: py-data.csv [hunk 2]
```diff
@@ -210,8 +210,8 @@ https://github.com/daknuett/ljson,1d3dc13001d9d2f61bcfbf9d5c5b16d44fb8076e,test/
 https://github.com/daknuett/ljson,1d3dc13001d9d2f61bcfbf9d5c5b16d44fb8076e,test/test_ljson_mem.py::test_contains,OD-Vic,,,
 https://github.com/daknuett/ljson,1d3dc13001d9d2f61bcfbf9d5c5b16d44fb8076e,test/test_ljson_mem.py::test_edit,OD-Vic,,,
 https://github.com/daknuett/ljson,1d3dc13001d9d2f61bcfbf9d5c5b16d44fb8076e,test/test_ljson_mem.py::test_unique_check,NIO,Accepted,https://github.com/daknuett/ljson/pull/4,
-https://github.com/daliclass/rxpy-backpressure,b1a7bddfb370884135d6f2e7d9a3dd332a90d7cc,tests/test_sized_buffer_backpressure_strategy.py::TestDropBackPressureStrategy::test_on_next_drop_new_message_when_buffer_full,OD-Vic,,,
-https://github.com/daliclass/rxpy-backpressure,b1a7bddfb370884135d6f2e7d9a3dd332a90d7cc,test_sized_buffer_backpressure_strategy.py::TestDropBackPressureStrategy::test_when_on_next_buffer_following_messages,OD,,,https://github.com/daliclass/rxpy-backpressure/issues/4
+https://github.com/daliclass/rxpy-backpressure,b1a7bddfb370884135d6f2e7d9a3dd332a90d7cc,tests/test_sized_buffer_backpressure_strategy.py::TestDropBackPressureStrategy::test_on_next_drop_new_message_when_buffer_full,OD-Vic,RepoArchived,,
```
Comment (xieyt2000) [inline py-data.csv:213]: I don't think this repo is archived?

**Review by xieyt2000 — CHANGES_REQUESTED** [review]
Please double check:
Some repos aren't archived. Did you find them manually or use some scripts?
For `MovedOrRenamed`, add new rows for the renamed tests
For `Deleted`, try to find when it was deleted

File: py-data.csv (hunk 2 above)
Comment (blazyy) [reply py-data.csv:213]: It was at the time of the commit, I think they have unarchived it after. I will update this.

File: py-data.csv (hunk 1 above)
Comment (blazyy) [reply py-data.csv:139]: Added a new row for the renamed test- I noticed there was a second duplicate instance of the same test. I've removed that row. Please let me know if that's alright

File: py-data.csv [hunk 3]
```diff
@@ -35,8 +35,8 @@ https://github.com/aluttik/kitsune,43824cccf46f433a71b30a7febc0e3500b831067,test
 https://github.com/AshtonUPS/Py-MI-PS,e43c2b4df2b59362c147024840bf541bb767ffd7,src/PyMIPS/tests/register_test.py,NIO,Open,https://github.com/AshtonUPS/Py-MI-PS/pull/23,
 https://github.com/AshtonUPS/Py-MI-PS,2d22327c75bac1b58a4804a61e7a703ecc5ba978,src/PyMIPS/tests/register_test.py::test_setting_to_num,NIO,Opened,https://github.com/AshtonUPS/Py-MI-PS/pull/22,
 https://github.com/AshtonUPS/Py-MI-PS,2d22327c75bac1b58a4804a61e7a703ecc5ba978,src/PyMIPS/tests/register_test.py::test_setting_to_num,OD-Vic,,,
-https://github.com/attzonko/mmpy_bot,4faa19a9e04c75af94c45c24dbd0c9d9ecb867d4,tests/unit_tests/test_scheduler.py::test_add_onetime_job_without_trigger_time,OD-Vic,,,
-https://github.com/attzonko/mmpy_bot,4faa19a9e04c75af94c45c24dbd0c9d9ecb867d4,tests/unit_tests/test_scheduler.py::test_add_onetime_job_with_trigger_time,OD-Vic,,,
+https://github.com/attzonko/mmpy_bot,4faa19a9e04c75af94c45c24dbd0c9d9ecb867d4,tests/unit_tests/test_scheduler.py::test_add_onetime_job_without_trigger_time,OD-Vic,Deleted,,
```
[the remark this answers is not in the document]
Comment (blazyy) [reply py-data.csv:38]: 8bba73afdd0dc3ce01c8b0599e59aa5688faf281

**Comment by darko-marinov** [comment]
`DeveloperWontFix` is when you don't even open a PR. I'll let @xieyt2000 inspect the details. Not all `Rejected` are treated the same.

File: py-data.csv (hunk 3 above)
[the remark this answers is not in the document]
Comment (xieyt2000) [reply py-data.csv:38]: Please add that in the `Notes` column

File: py-data.csv (hunk 1 above)
Comment (xieyt2000) [reply py-data.csv:139]: That's alright, seems like the two tests are duplicated (Although the repo URL isn't the same, they'll direct to the same repo)

File: py-data.csv [hunk 4]
```diff
@@ -324,7 +324,7 @@ https://github.com/earthreader/libearth,71d2f3246eb0b6a3668a64e2242616d20b1d7dcd
 https://github.com/earthreader/libearth,71d2f3246eb0b6a3668a64e2242616d20b1d7dcd,tests/stage_test.py::test_get_dir_route,OD-Vic,,,
 https://github.com/earthreader/libearth,71d2f3246eb0b6a3668a64e2242616d20b1d7dcd,tests/stage_test.py::test_set_dir_route,NIO,,,
 https://github.com/econ-ark/HARK,c9138af45fd0a0d97c718c8558f3ea8d9f2ae820,HARK/ConsumptionSaving/tests/test_ConsAggShockModel.py::testAggShockMarkovConsumerType::test_agent,OD-Vic,,,
-https://github.com/econ-ark/HARK,c9138af45fd0a0d97c718c8558f3ea8d9f2ae820,HARK/ConsumptionSaving/tests/test_ConsAggShockModel.py::testKrusellSmith::test_agent,OD-Vic,,,
+https://github.com/econ-ark/HARK,c9138af45fd0a0d97c718c8558f3ea8d9f2ae820,HARK/ConsumptionSaving/tests/test_ConsAggShockModel.py::testKrusellSmith::test_agent,OD-Vic,MovedOrRenamed,,Test renamed to KrusellSmithAgentTestCase::test_agent
```
Comment (xieyt2000) [inline py-data.csv:327]: For every `MovedOrRenamed` test, please add new rows for the renamed one

File: py-data.csv [hunk 5]
```diff
@@ -1351,7 +1351,7 @@ https://github.com/Teamworksapp/pydbvolve,e18bb8cf98997c52de0cf30a8e106c480ba0bb
 https://github.com/Teamworksapp/pydbvolve,e18bb8cf98997c52de0cf30a8e106c480ba0bb57,tests/unittests/test_00_initialization.py::test_11_log_file,NIO,Opened,https://github.com/Teamworksapp/pydbvolve/pull/14,
 https://github.com/Teamworksapp/pydbvolve,e18bb8cf98997c52de0cf30a8e106c480ba0bb57,tests/unittests/test_00_initialization.py::test_13_pre_config,OD-Vic,,,
 https://github.com/Teamworksapp/pydbvolve,e18bb8cf98997c52de0cf30a8e106c480ba0bb57,tests/unittests/test_01_migration_table.py::test_02_check_migration_table_bad_structure,OD-Brit,,,
-https://github.com/Tedyst/HikLoad,b3eb7be65bd34040992e1b7b15f23ff4ce2e8dc4,tests/test_config.py::test_envvar,OD-Brit,,,
+https://github.com/Tedyst/HikLoad,b3eb7be65bd34040992e1b7b15f23ff4ce2e8dc4,tests/test_config.py::test_envvar,OD-Brit,Deleted,,
```
Comment (xieyt2000) [inline py-data.csv:1354]: For every `Deleted`, please try to find in which commit were they deleted and add in the `Notes`

**Review by xieyt2000 — CHANGES_REQUESTED** [review]

**Comment by blazyy** [comment]
@xieyt2000 Added all the changes you requested, let me know if I missed anything

**Review by xieyt2000 — APPROVED** [review]
LGTM

**Comment by darko-marinov** [comment]
Accepting per Yuntong's approval. Please open smaller PRs in the future.

**Pull request merged**